### PR TITLE
KK-696 | Fix unauthorized page

### PR DESCRIPTION
**Pull request by Jokinen**
## Description

Due to earlier permission changes, the permission check made within the UI was no longer functioning properly.

Since the change, an admin profile has been returned for all users. Before the change the profile was only returned for users that had permissions to access a project.

The styles for the unauthorized page had also been broken. Likely when `react-admin` was updated to a  newer version.

## Context

[KK-696](https://helsinkisolutionoffice.atlassian.net/browse/KK-696)

## How Has This Been Tested?

I've added an e2e test that checks the unauthorized procedure.

## Manual Testing Instructions for Reviewers

As a user with permissions to access the admin

1. Login
2. Expect to be logged in

As a user without permission to access the admin

1. Login
2. Expect to be taken into the unauthorized page

### CODE REVIEW

File: browser-tests/utils/settings.ts [hunk 1]
```diff
@@ -1,24 +1,28 @@
 import * as dotenv from 'dotenv';
 
+// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+const get = require('lodash/get');
```
Comment (Jokinen) [inline browser-tests/utils/settings.ts:4]: The typescript setting Testcafe uses do not allow default imports and I didn't spend too much time debugging.

File: browser-tests/utils/settings.ts [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+
 dotenv.config();
 
-export const testUsername = (): string => {
-  if (!process.env.BROWSER_TESTS_UID) {
-    throw new Error('No BROWSER_TESTS_UID specified.');
-  }
-  return process.env.BROWSER_TESTS_UID;
-};
+function getOrError(variableName: string) {
```
Comment (Jokinen) [inline browser-tests/utils/settings.ts:8]: Sorry for the noise that this refactor causes.

**Review by tuovinensanttu — APPROVED** [review]
Tested this locally and it works! Could you update `CHANGELOG.md` as well?

**Comment by Jokinen** [comment]
Done!

**Pull request merged**